ts/nni_manager/core/nniTensorboardManager.ts

```diff
@@ -114,9 +114,9 @@ class NNITensorboardManager implements TensorboardManager {
     }
 
     private setTensorboardVersion(): void {
-        let command = `python3 -c 'import tensorboard ; print(tensorboard.__version__)'`;
+        let command = `python3 -c 'import tensorboard ; print(tensorboard.__version__)' 2>&1`;
         if (process.platform === 'win32') {
-            command = `python -c "import tensorboard ; print(tensorboard.__version__)"`;
+            command = `python -c "import tensorboard ; print(tensorboard.__version__)" 2>&1`;
```

Comment:
1 means stdout, you redirect stderr message to stdout? nnictl also collect stdout right?

Reply:
maybe " >/dev/null 2>&1", not sure whether it works in windows...

Reply:
no, stdout will be collected by nnimanager

Reply:
ok

Reply:
tensorboard manager needs the stdout of this command to check the version, so `>/dev/null` will not work?